lib/susedistribution.pm

```diff
@@ -434,8 +434,8 @@ sub init_consoles {
         }
     }
 
-    if (get_var('SUT_IP') || get_var('VIRSH_GUEST')) {
-        my $hostname = get_var('SUT_IP', get_var('VIRSH_GUEST'));
+    if (get_var('SUT_IP') || get_var('VIRSH_GUEST') || get_var('ZVM_HOST')) {
+        my $hostname = get_var('SUT_IP', get_var('VIRSH_GUEST', get_var('ZVM_HOST')));
```

Comment:
These two lines should be reverted since the console gets added by another code branch below.